<launch>
    <node pkg="pointcloud_utils" type="plane_points_generator" name="plane_points_pub" required="true" output="screen">
    	<param name="lidar_topic"	value="/points" />
    	<param name="frame_id" 		value="lidar" />
    	<param name="point_density" value="40"/>
    	<param name="x_bound"		value="5"/>
		<param name="y_bound"		value="5" />
		<param name="z_height" 		value="1" />
		<param name="roll"			value="0.1" /> #was 0.785398 (45 deg)
		<param name="pitch" 		value="0.1" /> #was 0.785398
		<param name="noise" 		value="0.025" />
		<param name="pub_rate" 		value="10" />
		<param name="rolling_sinusoid" 		value="true" />
		<param name="pitching_sinusoid" 	value="true" />
		<param name="sinusoid_roll_rate" value="5" />
		<param name="sinusoid_pitch_rate" value="2" />
		<param name="sinusoid_pitch_phase_shift" value="1" />
		<param name="sinusoid_roll_phase_shift" value="0" />
    </node>
</launch> 